src/components/ConnectionRequired.tsx

```diff
@@ -96,6 +96,9 @@ const ConnectionRequired: FunctionComponent<ConnectionRequiredProps> = ({
                         }
                         const systemInfo = await infoResponse.json();
                         if (!systemInfo?.StartupWizardCompleted) {
+                            // Update the current ApiClient
+                            // TODO: Is there a better place to handle this?
+                            ServerConnections.setLocalApiClient(firstConnection.ApiClient);
```

Comment:
This is something the `appRouter` does that we missed when porting the logic over. Without it you can end up with an exception because the current api client is null after navigating to the startup wizard.